src/lab12.py

```diff
@@ -732,92 +734,9 @@ def handle_quit(self):
             tab.task_runner.set_needs_quit()
         sdl2.SDL_DestroyWindow(self.sdl_window)
 
-@wbetools.patch(Chrome)
```

Comment:
Why delete this? I thought the change was using `active_tab_url` instead of just `tab.url`, which is a potential race?

Reply:
Oh, weird, we had two. Huh. Ok, got it.